### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by owenhallgren**
## *Purpose:*
Adds basic testing to main page/movie details page
*What does this merge do? Check all that apply.*

- [ ] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*

*Briefly, but clearly describe what you did, and explain any new code.*
This adds testing to the main page, confirms that movies are loaded in. This tests for a movie details page popping up when a movie is clicked. This tests the back button to confirm you are taken back to the home page. 

## *Instructions for reviewer:*

*Where should the reviewer start?*
The reviewer should look over the tater_spec.js testing file and look for more potential tests

*How should this be manually tested?*
Open cypress and run the tests

*Any background context you want to provide?*
N/A
*What are the relevant tickets?*

## *Issues:*

*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: cypress/integration/examples/taters_spec.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ describe('home page', () => {
     .get('p').first().contains('Average Rating: 6.1 🥔 \'s')
     .get('p').contains('Release Date: 2020-09-29')
   })
+
+  it('Should display movie details', () => {
+    cy.visit('http://localhost:3000')
+    .get('.movie-card').first().click()
+    .get('.movie-details')
+    .get('.back-drop').should('have.attr', 'src').should('include', 'https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg')
```
Comment (Max9545) [inline cypress/integration/examples/taters_spec.js:20]: Add a intercept later

File: cypress/integration/examples/taters_spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+describe('home page', () => {
+  it('Should it should have a header', () => {
+    cy.visit('http://localhost:3000')
+    .get('h1').contains('Rotten Taters 🥔')
+  })
+
+  it('Should display movies', () => {
+    cy.visit('http://localhost:3000')
```
Comment (letakeane) [inline cypress/integration/examples/taters_spec.js:8]: I'm guessing this is before refactoring to use the actual API. You probably already caught this, but you'll need an intercept here eventually.

File: cypress/integration/examples/taters_spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    .get('.movie-card').first()
+    .get('h3').contains('Money Plane')
+    .get('img').should('have.attr', 'src').should('include', 'https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg')
```
Comment (letakeane) [inline cypress/integration/examples/taters_spec.js:11]: Great specificity! Love this test.

**Review by letakeane — COMMENTED** [review]
Nice start to testing. Great job being specific with what you expect of the UI!

**Pull request merged**